src/main/java/bstore/bookstore/model/Book.java

```diff
@@ -7,17 +7,15 @@
 import jakarta.persistence.Id;
 import jakarta.persistence.Table;
 import java.math.BigDecimal;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
 import lombok.Data;
-import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
 
 @Entity
-@Table(name = "books")
 @Data
```

Comment:
Don't use `@Data` for entities
```suggestion
@Getter
@Setter
```